Makefile

```diff
@@ -1,22 +1,23 @@
-default: build/universal-robots
+.PHONY: module.tar.gz ur5e-sim format test clean clean-all docker docker-build docker-amd64 docker-upload appimages docker-arm64-ci docker-amd64-ci
```

Comment:
.PHONY means `doesn't produce a file or directory with the name of the target` all these commands are phony